requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.8.0
-numpydoc==0.4
-networkx==1.8.1
+numpy>=1.8.0
+numpydoc>=0.4
+networkx>=1.8.1
```

Comment:
It is a nice statement (although @astaric might disagree). When it's not this file, it's _setup.py_. Whatever way is fine by me, but keeping this file while it's being unused is woefully misleading.